docs/api/attributions/guided_backpropagation.md

```diff
@@ -1,9 +1,34 @@
 # Guided Backpropagation
 
-<sub><img src="https://upload.wikimedia.org/wikipedia/commons/d/d0/Google_Colaboratory_SVG_Logo.svg" width="20"></sub>[ View colab tutorial](https://colab.research.google.com/drive/19eB3uwAtCKZgkoWtMzrF0LTJ-htF_KE7?authuser=1) | <sub><img src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png" width="20"></sub>[ View source](https://github.com/deel-ai/xplique/blob/master/xplique/attributions/guided_backpropagation.py) | 📰 [ See paper](https://arxiv.org/abs/1412.6806)
+<sub>
+    <img src="https://upload.wikimedia.org/wikipedia/commons/d/d0/Google_Colaboratory_SVG_Logo.svg" width="20">
+</sub>[View colab tutorial](https://colab.research.google.com/drive/19eB3uwAtCKZgkoWtMzrF0LTJ-htF_KE7) |
+<sub>
+    <img src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg" width="20">
+</sub>[View source](https://github.com/deel-ai/xplique/blob/master/xplique/attributions/guided_backpropagation.py) |
+📰 [Paper](https://arxiv.org/abs/1412.6806)
+
+Guided-backprop is one the first attribution method and was proposed in 2014.
```

Comment:
Guided backpropagation is one of the first (we are writing full sentences I think it is better to not use shortcut for words)